.github/workflows/release.yml

```diff
@@ -57,10 +65,8 @@ jobs:
       - name: Publish wheel and source distribution
```

Comment:
comment is out of date

Reply:
What do you mean?

Reply:
It does not longer publish the source here.

Reply:
It still does on pypi

Reply:
Oh, right...